INSTALL-macos.md

```diff
@@ -0,0 +1,89 @@
+# Installing the CircleCI Runner on macOS
+
+## Create a runner configuration
+Choose a user to run the CircleCI agent as. These instructions will refer to the selected user as `USERNAME`.
+
+Complete the below template, with the various capitalised parameters filled in, and save as `launch-agent-config.yaml`.
+```yaml
+api:
+    auth_token: AUTH_TOKEN
+runner:
+    name: NAME
+    command_prefix : ["sudo", "-niHu", "USERNAME", "--"]
+    working_directory: /tmp/%s
+    cleanup_working_directory: true
+logging:
+    file: /Library/Logs/com.circleci.runner.log
+```
+
+## Install the runner configuration
+Create a directory as root to hold the runner configuration: 
+```bash
+sudo mkdir -p '/Library/Preferences/com.circleci.runner'
+```
+
+Copy the previously created `launch-agent-config.yaml` into the directory:
+```bash
+sudo cp 'launch-agent-config.yaml' '/Library/Preferences/com.circleci.runner/launch-agent-config.yaml'
+```
+
+## Install the launchd .plist
+
+Copy the following to `/Library/LaunchDaemons/com.circleci.runner.plist`, owned by root, with permissions `644`:
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Label</key>
+        <string>com.circleci.runner</string>
+
+        <key>Program</key>
+        <string>/usr/local/bin/circleci-launch-agent</string>
+
+        <key>ProgramArguments</key>
+        <array>
+            <string>circleci-launch-agent</string>
+            <string>--config</string>
+            <string>/Library/Preferences/com.circleci.runner/launch-agent-config.yaml</string>
+        </array>
+
+        <key>RunAtLoad</key>
+        <true/>
+
+        <key>KeepAlive</key>
+        <dict>
+          <key>SuccessfulExit</key>
+          <false/>
+        </dict>
+
+        <key>ProcessType</key>
+        <string>Interactive</string>
+
+        <key>ThrottleInterval</key>
+        <integer>30</integer>
+
+        <key>StandardOutPath</key>
+        <string>/dev/null</string>
+
+        <key>StandardErrorPath</key>
+        <string>/dev/null</string>
```

Comment:
if we omit these, what's the default?

Reply:
I'm not totally sure, but other similar services are doing this, so I followed suit. I wanted to be sure the logs weren't going somewhere else, or building up in some memory buffer.